src/System.Drawing.Common/tests/BufferedGraphicsContextTests.cs

```diff
@@ -254,9 +254,11 @@ public void Finalize_Invoke_Success()
         {
             // Don't allocate anything as this would leak memory.
             // This makes sure than finalization doesn't cause any errors or debug assertions.
-            using (var context = new BufferedGraphicsContext())
-            {
-            }
+            var context = new BufferedGraphicsContext();
+            context = null;
+
+            GC.Collect();
+            GC.WaitForPendingFinalizers();
```

Comment:
This is not guaranteed, and in fact unlikely to actually run the finalizer.  If you want to significantly increase the chances, move the allocation into a non-inlined void helper.  You can see this with a simple repro like:
```C#
using System;
using System.Runtime.CompilerServices;

class Program
{
    ~Program() => Console.WriteLine("finalized");

    [MethodImpl(MethodImplOptions.NoInlining)]
    static void Allocate() => new Program();

    static void Main()
    {
        var p = new Program();
        p = null;
        //Allocate();
        GC.Collect();
        GC.WaitForPendingFinalizers();
        Console.ReadLine();
    }
}
```

Reply:
OK - I updated the test.